### PR TITLE
Fix displaying problems for Bilibili players from #572 & Fix song thumb loading problems from #347

### PR DESCRIPTION
**Pull request by AgFlore**
For the [problems](https://github.com/VocaDB/vocadb/pull/572#issuecomment-605606181) with the layout of Bilibili players:

> Not showing correctly in playlists (player became in the same line with the title. Check for example https://vocadb.net/L/8246)

Fixed this by adding a style attribute of `width:max-content` to the wrapping `<div>`.

> Not showing correctly in the homepage ranking (the size set for the player seemed to be ignored)
There were multiple problems with the layout in the homepage ranking: 

First, the size set for the `<iframe>` was covered by the stylesheet, so we had to re-address it in the `style` manually. 
Next, setting the `position` as `absolute` (from the stylesheet as well!) would prevent the wrapper from hiding the manually masked content, so I set is as `relative` as in other pages. 
Then we arrrived at the current fix; it's still not perfect, as the player is 25px too low (due to the 25px `padding-top` from `.pvViewerContent .pv-embed-wrapper`'s stylesheet), which ends up to cover the songInfoButton. But I think that's tolerable, as one can check the page for the song by clicking the title, so giving up the button was not that fatal.
Anyway that's not a elegant or perfect fix, still looking forward to getting it better; maybe one need to do some work on the layout/stylesheet of the page as well to fix this completely.

For the issue #347 , one simply add `referrerpolicy="same-origin"` to everywhere a song thumb may be loaded, and hopefully this can be settled. Seems that `referrerpolicy` cannot be used as a `style` attribute, so I cannot fix it in the stylesheets, and I need to add it everywhere manually.

### CODE REVIEW

File: VocaDbWeb/Views/Album/Details.cshtml [hunk 1]
```diff
@@ -356,7 +356,7 @@
 			</h3>
 			<div class="media" data-bind="with: personalDescription">
 				<a data-bind="visible: !author.isEmpty() && author.entry().mainPicture, attr: { href: vdb.utils.EntryUrlMapper.details('Artist', author.id()) }" class="pull-left">
-					<img class="coverPicThumb" alt="Thumb" data-bind="attr: { src: !author.isEmpty() && author.entry().mainPicture ? author.entry().mainPicture.urlSmallThumb : null }" />
+					<img class="coverPicThumb" alt="Thumb" data-bind="attr: { src: !author.isEmpty() && author.entry().mainPicture ? author.entry().mainPicture.urlSmallThumb : null }" referrerpolicy="same-origin" />
```
Comment (riipah) [inline VocaDbWeb/Views/Album/Details.cshtml:359]: Why would you add it here? Album images are always from VocaDB domain

File: VocaDbWeb/Views/Search/Partials/_TagSearchList.cshtml [hunk 2]
```diff
@@ -34,7 +34,7 @@
 			<tr>
 				<td style="width: 80px">
 					<a data-bind="visible: $data.mainPicture && mainPicture.urlSmallThumb, attr: { href: vdb.utils.EntryUrlMapper.details_tag(id, urlSlug), title: $data.additionalNames }" href="#">
-						<img data-bind="attr: { src: ($data.mainPicture ? mainPicture.urlSmallThumb : '') }" title="Cover picture" class="coverPicThumb img-rounded" />
+						<img data-bind="attr: { src: ($data.mainPicture ? mainPicture.urlSmallThumb : '') }" title="Cover picture" class="coverPicThumb img-rounded" referrerpolicy="same-origin" />
```
Comment (riipah) [inline VocaDbWeb/Views/Search/Partials/_TagSearchList.cshtml:37]: But surely these images are never off-site?

Comment (AgFlore) [reply VocaDbWeb/Views/Search/Partials/_TagSearchList.cshtml]: Yeps, I was doing quite blindly when adding the referrerpolicy tag as I was not 100% definite about what each picture actually is. Supposingly only song thumbs need this; but with the `same-origin` attribution, images already from our own domain wouldn't be effected, so over-tagging would only be a tidiness issue. I've removed the two unnecessary occurances you mentioned.

**Comment by riipah** [comment]
All right, thank you.

**Pull request merged**